plugin-modernizer-core/src/test/resources/plugins.txt

```diff
@@ -0,0 +1,3 @@
+jobcacher-plugin
```

Comment:
This is the repo name, not the plugin name.

I think we are wrongly using pluginName with -plugin suffix but in fact it's the repo name.

Perhaps we should clarifiy this in the tool before this PR.

Passing the plugin short name and getting the SCM url from the plugin center ? Or just ammending the `-plugin` to create the repo ? But I'm not sure it will work 100% of time

Reply:
Better safe than sorry, we should make it super clear in the documentation, the comments, and the code. 👍 
Thanks for catching that early!